BlackNectar/BlackNectar/StoresDataStruct.swift

```diff
@@ -91,16 +101,15 @@ extension Address {
     
     init?(from storeDictionary: NSDictionary) {
         
-        guard let addressLineOne = storeDictionary ["address_line_1"] as? String else { return nil }
-        
-        let addressLineTwo = storeDictionary ["address_line_2"] as? String
+        guard let addressLineOne = storeDictionary["address_line_1"] as? String else { return nil }
```

Comment:
Got it. Much cleaner. This correction has been completed in another branch.
Done.